<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>404</width>
    <height>278</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>拓展</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4" stretch="1,1,6">
   <property name="spacing">
    <number>20</number>
   </property>
   <property name="leftMargin">
    <number>20</number>
   </property>
   <property name="topMargin">
    <number>20</number>
   </property>
   <property name="rightMargin">
    <number>20</number>
   </property>
   <property name="bottomMargin">
    <number>20</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_9" stretch="5,1">
     <property name="spacing">
      <number>10</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,10">
       <item>
        <widget class="QLabel" name="Label">
         <property name="text">
          <string>求导</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="line1">
         <property name="placeholderText">
          <string>请输入表达式，e写E，派写pi，sin x写sin(x)</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="qiudao">
       <property name="text">
        <string>求导</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_10" stretch="5,1">
     <property name="spacing">
      <number>10</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="2,1,1,9">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>极限</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>x→</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="line2"/>
       </item>
       <item>
        <widget class="QLineEdit" name="line3">
         <property name="placeholderText">
          <string>无穷oo，负无穷-oo</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="jixian">
       <property name="text">
        <string>极限</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_11" stretch="5,1">
     <property name="spacing">
      <number>10</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="1,10">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>积分</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,1">
         <property name="spacing">
          <number>50</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="2,1,4,2,4">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>不定积分</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>∫</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="line4"/>
           </item>
           <item>
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>dx  =</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="line5"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="1,7">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>定积分</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="1,8">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,1">
               <item>
                <widget class="QLabel" name="label_7">
                 <property name="text">
                  <string>∫</string>
                 </property>
                </widget>
               </item>
               <item>
                <layout class="QVBoxLayout" name="verticalLayout" stretch="1,1">
                 <item>
                  <widget class="QLineEdit" name="line6"/>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="line7"/>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="4,1,4">
               <item>
                <widget class="QLineEdit" name="line8"/>
               </item>
               <item>
                <widget class="QLabel" name="label_9">
                 <property name="text">
                  <string>dx  =</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="line9"/>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1,1">
       <property name="spacing">
        <number>50</number>
       </property>
       <item>
        <widget class="QPushButton" name="budingjifen">
         <property name="text">
          <string>不定积分</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="dingjifen">
         <property name="text">
          <string>定积分</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
